scripts/process-image-manifests

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+# This script extracts the most recent manifests from an image containing a manifests bundle and processes it into a version that works with this operator.
+#
+# Example:
+#
+#$ ./process-image-manifests registry-proxy.engineering.redhat.com/rh-osbs/3scale-amp2-3scale-rhel7-operator-metadata:1.11.0-2
+#$ tree manifests/registry-proxy.engineering.redhat.com/rh-osbs/ -L 2
+#manifests/registry-proxy.engineering.redhat.com/rh-osbs/
+#└── 3scale-amp2-3scale-rhel7-operator-metadata:1.11.0-2
+#    ├── 0.3.0
+#    ├── 0.4.0
+#    ├── 0.4.1
+#    ├── 0.4.2
+#    ├── 0.5.0
+#    └── 3scale-operator.package.yaml
+
+set -e
+
+WORK_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+MANIFESTS_DIR=${WORK_DIR}/../manifests
+MANIFEST_TMP_DIR=${MANIFESTS_DIR}/temp
+IMAGE=$1
+FOLDER_PRODUCT_NAME=3scale
```

Comment:
Could we pass this from the pipeline. Would save us having to come up with some logic in here to figure out which product it's for. We'd know that from the UMB message.

Reply:
If that is easier, then yes